pkg/log/log.go

```diff
@@ -3,33 +3,21 @@ Package log provides support for logging to stdout and stderr.
 
 Log entries will be logged in the following format:
 
-    timestamp hostname tag[pid]: SEVERITY Message
```

Comment:
These changes seem unrelated to the Makefile.common change. What is the reason for the change?

Reply:
Lint picked these out. The `tag` variable is not being used.